examples/cpp/hello_tf.cpp

```diff
@@ -0,0 +1,113 @@
+/*******************************************************************************
+ * Copyright 2017-2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+
+#include "tensorflow/core/framework/graph.pb.h"
+#include "tensorflow/core/framework/op.h"
+#include "tensorflow/core/framework/tensor_types.h"
+#include "tensorflow/core/graph/algorithm.h"
+#include "tensorflow/core/graph/default_device.h"
+#include "tensorflow/core/graph/graph.h"
+#include "tensorflow/core/graph/graph_constructor.h"
+#include "tensorflow/core/platform/env.h"
+
+#include "tensorflow/cc/client/client_session.h"
+#include "tensorflow/cc/ops/standard_ops.h"
+#include "tensorflow/core/framework/tensor.h"
+#include "tensorflow/core/public/session.h"
+
+#include "ngraph_bridge/ngraph_api.h"
+#include "ngraph_bridge/ngraph_backend_manager.h"
+
+using namespace std;
+
+// Prints the available backends
+void PrintAvailableBackends() {
+  // Get the list of backends
+  auto supported_backends =
+      tensorflow::ngraph_bridge::BackendManager::GetSupportedBackendNames();
+  vector<string> backends(supported_backends.begin(), supported_backends.end());
+
+  cout << "Available backends: " << endl;
+  for (auto& backend_name : backends) {
+    cout << "Backend: " << backend_name << std::endl;
+  }
+}
+
+// Sets the specified backend. This backend must be set BEFORE running
+// the computation
+tensorflow::Status SetNGraphBackend(const string& backend_name) {
+  // Select a backend
+  tensorflow::Status status =
+      tensorflow::ngraph_bridge::BackendManager::SetBackendName(backend_name);
```

Comment:
This style of backend setting does not work in grappler style builds I think. For grappler builds, we have to use the rewriter config